components/pedigree/resources/src/main/resources/pedigree/view/personHoverbox.js

```diff
@@ -23,8 +23,22 @@ define([
     var PersonHoverbox = Class.create(AbstractHoverbox, {
 
         initialize: function($super, personNode, centerX, centerY, nodeShapes) {
-            var radius = PedigreeEditorParameters.attributes.personHoverBoxRadius;        
-            $super(personNode, -radius, -radius, radius * 2, radius * 2, centerX, centerY, nodeShapes);                
+            var radius = PedigreeEditorParameters.attributes.personHoverBoxRadius;
+            $super(personNode, -radius, -radius, radius * 2, radius * 3, centerX, centerY, nodeShapes);
+        },
+
+        /**
+         * Returns the gray hover box height extension
+         *
+         * @method getLabelsOffset
+         * @return {int} 0 by default or PedigreeEditorParameters.attributes.personHoverBoxRadius if we display alive and well radio buttons
+         */
+        getLabelsOffset: function() {
+            if (this.getNode().getLifeStatus() == "alive" || this.getNode().getLifeStatus() == "deceased") {
+                return PedigreeEditorParameters.attributes.personHoverBoxRadius - 20;
```

Comment:
instead of some non-obvious computation better add a new parameter to PedigreeEditorParameters.attributes with a comment explaining what it is (which should be done in any case, so that if e.g. font size for labels is changed there the howerbox can be sized accordingly at the same place)